src/node/build/index.ts

```diff
@@ -669,29 +669,30 @@ export async function ssrBuild(
 
   return build({
     outDir: 'dist-ssr',
+    emitIndex: false,
+    emitAssets: true, // needed for manifest.json
+    cssCodeSplit: false,
+    preserveEntrySignatures: 'strict', // so generated files can be consumed by server
```

Comment:
This should be in `rollupInputOptions`

Reply:
Ya I don't think anyone has as it doesn't work lol

Reply:
It would be awesome if you added an SSR playground. :)
https://github.com/vitejs/vite/tree/master/playground

Reply:
@aleclarson I'd love to help out with it but I'll need to make some core changes as I hit about 10 dead-ends earlier today.

Reply:
Honestly, I don't know why Vue SSR can't work like `ts-node` there are probably good reasons but there is zero documentation on this subject.